web/urls.py

```
@@ -6,6 +6,8 @@
 from django.urls import include, path
 
 from . import admin_views, peer_challenge_views, quiz_views, views
+from web import views
```

Ruff (0.8.2)

Redefinition of unused `views` from line 8

(F811)